lib/private/User/Session.php

```diff
@@ -211,6 +211,15 @@ public function setUser($user) {
 		$this->activeUser = $user;
 	}
 
+	/**
+	 * Temporarily set the currently active user without persisting in the session
+	 *
+	 * @param IUser|null $user
+	 */
+	public function setActiveUser(?IUser $user): void {
```

Comment:
```suggestion
	public function setVolatileActiveUser(?IUser $user): void {
```

Consideration?? (or a more indicative name) Given what the function actually does?

Reply:
Good idea, adjusted